components/centraldashboard/app/stackdriver_metrics_service.ts

```diff
@@ -194,4 +194,11 @@ export class StackdriverMetricsService implements MetricsService {
     }
     return this.clusterName;
   }
+
+  getChartsLink(): MetricsInfo {
+    return {
+      resourceChartsLink: `https://app.google.stackdriver.com/kubernetes?project=${this.projectId}`,
```

Comment:
I dont think this will work if we block internet access, we might have to take out this feature, or make sure that this stackdriver app is accessible from the zone, if we want to keep it.

Reply:
Added comment to https://jirab.statcan.ca/browse/BTIS-623 which is tracking possible breaking changes